observing_examples.ipynb

```diff
@@ -0,0 +1,950 @@
1DreWb6Nx2esZfbaXbRLS+aWM3sxeUxBRPdJOOd47/Pt3PfuKpZt3osZDMjNYEyvjpzZrxOnFnSgVZJqKCJBUxLxxFoSOfwh/H9fX866neXkZaZx7ejuXDUin3Zp0dP34Jxj/obdzCjeyczVO5i/fg+VNbW0aRXqUxndqyMjenSgf04GiQn13ZgnIn5SEvHEUhJZU7qfu6cv58OVpfTu1Jbbv9WXbw3oTFIM9C2UV1Yzs3gnH6zczkerSinZfRCA9JQk8ju2JrddKnmZaVwyrCtDumUGHK1I7FMS8cRCEinZXc797xfzclEJacmJ/OicPkweUxDTHdOb9hxk7tpdFK3fzaY9B9lSVvFl38rpfbL417P7MKJHh6DDFIlZSiKeaE4i5ZXV3PPmCp6fswHDmDiiGz84uzed0lODDi0Q+w9V88xn63n0kzXs2F9JRmoS3Tq0Jr9DawZ3y+S03lkMyM0gQc1fIidMScQTrUlk056D3PDkPD7fuperR+Rz69m9yW2XFnRYEeFgZQ1/W7iJ5Zv3snF3Oet3lrN2xwEA2rdOZmSPjpzaowMje3TgpFz1qYg0hV9JRGNdtIC563bx/aeLqKyu5S+TT+Ws/p2CDimipLVKZOKII4ez2b63ghmrd/DpFzuZs24n/1y2FYAu7VK54tRuXFHYjS6ZSsIiQVNNxEe1tY6pn6zhd2+tJL9Da6ZeW0jvTm2DDisqbSk7yGdrdvLq/E188sUOEgwmDMnj5+P7x21zoMjxUHOWJ1qSyPZ9Fdz+4iI++WIH55+cwz2XDYqqW3Yj2cZd5Tzz2Xoen7GOlOQEfnZePyaN7K5mLpEGKIl4oiGJzF23i5ufKWJfRTX/cdFAJo7ohpk+4JrbmtL93PX3ZXxavIP+Oenc5t0irX9rka/zK4nE7j2lAXm5qISrH/mM9NRkpt16GlePzNeHmk96Zrfl6SkjuP/qoRyqruXGp4v49gMzmLuuobnQRKQ5KYk0k9paxz1vruCnLy3i1IIOvHbLGPrlpAcdVswzMy4c1IV3fnIG/33ZIHbsr+TqRz7jvbABI0XEP0oizeSP73/BQx+t5uqR+Tz53REahLCFJSUmcMWp3Xjjh6dzUm4G33+miHeXK5GI+E1JpBks2riHP71fzLeHdOE/v31yTD95HunatU7m6SkjGZCbwc3PFvGOEomIr/Rpd4Iqqmr4yYsLyW6bwq8nnKz+jwjQLi2Zp6aMZECXdvzg2fl8+sWOoEMSiVlKIifo3n+uYE3pAX73L4N1C28EaZeWzJPXn0rP7Dbc8NQ8itbvDjokkZikJHIC3vt8G4/PWMd1Ywo4rU9W0OFIHZmtW/HUlBF0zkjh+sfnsGxzWdAhicQcJZEm+nzLXn74/AJOzsvg38b1DzocOYpO6ak8872RtE1J4vrH57J5z8GgQxKJKUoiTbB9XwVTnphLemoyj00+lbRWkTvzoEDX9q154rsjOFhZw5Qn57H/UHXQIYnEDCWR41RRVcMNTxWxu7yKRycX0jlD4zZFg76d07l/0jBWbdvHD59fQE1tdI3UIBKplESO033vrmLRxj384aohnJzXLuhw5Dh8o282v7p4IO+vCE1HLCInTkPBH4eNu8p5/NN1XDasK+cNzAk6HGmC74zqztrSA/xlxlqGdMtkwpC8oEMSiWqqiRyHe/+5goQE+Nl5/YIORU7AL8b3p7B7e37x6hLWlO4POhyRqKYk0khF63czffEWbjqjFznt1A8SzZISE/jjxKEkJyXwg+cWUFFVE3RIIlFLSaQRnHP8ZvpyOqWncNM3egYdjjSDLplp/P6KwXy+ZS+/ma7+EZGmUhJphOmLt7Bw4x5+el4/WrdSN1KsOLt/Z246oyfPzt7A3xduCjockaikJHIMldW1/O7tlfTPSeeyYV2DDkea2U/P68epBe35+atLKN6+L+hwRKKOksgx/HXuBtbvLOeOcf00/WoMSk5M4E8Th5GWnMj3n5nPAT2IKHJclEQacOBQNf/7XjEjCjpwVr9OQYcjPslpl8ofJw5ldel+fvHaEqJtymiRICmJNOAvn65lx/5D/Nv5/TXEe4wb2zuL287py98XbubZ2RuCDkckaiiJHMWuA5U8/PEazh3QmeHd2wcdjrSAH5zVmzP7ZXP3P5azuGRP0OGIRAVfk4iZjTOzlWZWbGZ31rM/38w+MLMFZrbYzMb7Gc/xeG3BJvYfqua2c/sGHYq0kIQE474rhpCdnsLNz8xnT3ll0CGJRDzfkoiZJQIPAOcDA4CJZjagTrF/B150zg0FrgL+7Fc8x2v64s2clJtB/5yMoEORFtS+TSsemDSM7fsquP3FReofETkGP2siI4Bi59wa51wl8AIwoU4ZBxz+lG4HbPYxnkbbuKucBRv2cNHg3KBDkQAM6ZbJL8afxHsrtvP6ki1BhyMS0fxMInnAxrD1Em9buF8B15hZCfAG8K8+xtNohz84LjylS8CRSFCuHV3AgNwM/uuNFRoWRaQBfiaR+m5nqts2MBF4wjnXFRgPPG1mX4vJzG40s3lmNq+0tNSHUI80ffFmBndtR37H1r6/l0SmxATj3y88iU17DvLYp2uDDkckYvmZREqAbmHrXfl6c9UU4EUA59wsIBX42mTlzrmpzrlC51xhdna2T+GGrN1xgKWb9nLRYNVC4t2YXlmcO6Azf/6gmO37KoIORyQi+ZlE5gJ9zKyHmbUi1HE+rU6ZDcA3AczsJEJJxP+qRgOmLwrlufGnqD9E4BfjT6Kyppb/eWtV0KGIRCTfkohzrhq4FXgL+JzQXVjLzOxuM7vYK3Y7cIOZLQKeB65zAd8OM33xFk4taE+XzLQgw5AIUZDVhuvGFPBi0UbmrN0VdDgiEcfX50Scc2845/o653o55/7T23aXc26at7zcOTfWOTfYOTfEOfe2n/Ecy6pt+1i5bR8XDlJTlnzlx+f0pWv7NO54eREHK9XJLhIupp9YP1Rdw+4Dx35grLqmlhfmbGDyX+aQ
```

Comment:
The reason I specify the unit in the comment instead is because the temperature is just used to query the correct model spectrum in the following lines. No calculations are done with it.